test/index.js

```diff
@@ -379,6 +381,54 @@ describe('Memory', function () {
         done();
     });
 
+    describe('stopping after set', function () {
```

Comment:
Sure, `parallel : false` makes total sense. I want to make sure that I understand your 'single test' point. By that do you mean a single `it` clause rather than a `describe` block with `before` and `after`? My concern with this approach is that in the event of a test failure the rest of the test cases could end up running without the `setTimeout` and `clearTimeout` changes being cleaned up. Given the current mock implementations, I don't expect there to be any noticable problems in this case, but allowing changes to the global state to possibly hang around seemed like a bad idea to me. Happy to do whatever you think is best though.